features/docker.feature

```diff
@@ -0,0 +1,71 @@
+@uses.config.contract_token
+Feature: Build docker images with ua services
+
+    @slow
+    @docker
+    @series.focal
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Build docker images with ua services
+        Given a `focal` machine with ubuntu-advantage-tools installed
+        When I have the `<container_release>` debs under test in `/home/ubuntu`
+        When I run `apt install -y docker.io` with sudo
+        When I create the file `/home/ubuntu/Dockerfile` with the following:
+        """
+        FROM ubuntu:<container_release>
+
+        <copy_local_deb>
+
+        RUN --mount=type=secret,id=ua-attach-config \
+            apt-get update \
+            && apt-get install --no-install-recommends -y <test_package_name> \
+            && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
+
+            && <install_ua_under_test> \
+
+            && ua attach --attach-config /run/secrets/ua-attach-config \
+
+            # Normally an apt upgrade is recommended, but we dont do that here
+            # in order to measure the image size bloat from just the enablement
+            # process
+            # && apt-get upgrade -y \
```

Comment:
I thought this was true at one point during testing but I'll try adding it back in

Reply:
The comment here is accurate